pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -22,6 +22,20 @@ package pulsar.proto;
 option java_package = "org.apache.pulsar.common.api.proto";
 option optimize_for = LITE_RUNTIME;
 
+message Schema {
+	required string name = 1;
+	required bytes version = 2;
+	required bytes schema_data = 3;
+    repeated KeyValue properties = 4;
+}
+
+message Tombstone {}
+
+message SchemaEntry {
```

Comment:
I assume this is part of a the schema registry part? Why is tombstone separate from the schema? You have schema_data as required? If tombstoning a schema (which I assume is deleting the schema) what should the data be? Couldn't schema data itself being empty denote the tombstone?